PythonStepTemplate.yml

```
@@ -0,0 +1,94 @@
+# STEPS LEVEL TEMPLATE:
+# Used to build Python
+# Reusable
+# Used to "hide" the additional variables specific to this SDK which shouldn't be set from a higher level, or
+#   shared from a multi-build pipeline like a publish
+
+parameters:
+- name: ApiSpecSource
+  displayName: ApiSpecSource
+  type: string
+  default: -apiSpecGitUrl https://raw.githubusercontent.com/PlayFab/API_Specs/master/
+- name: CommitMessage
+  displayName: CommitMessage
+  type: string
+  default: Automated build from ADO Pipeline
+- name: GitDestBranch
+  displayName: GitDestBranch
+  type: string
+  default: doNotCommit
+- name: SdkName
+  displayName: SdkName
+  type: string
+  default: PythonSDK
+- name: GitJSetupBranch
+  displayName: GitJSetupBranch
+  type: string
+  default: master
+- name: GitSdkGenBranch
+  displayName: GitSdkGenBranch
+  type: string
+  default: master
+- name: isVersioned
+  displayName: isVersioned
+  type: boolean
+  default: false
+- name: TestServer
+  displayName: TestServer
+  default: false
+- name: SelfTemplateResource
+  displayName: SelfTemplateResource
+  type: string
+  default: self
+
+steps:
+- checkout: JenkinsSdkSetupScripts
+  clean: true
+  path: s
+- checkout: API_Specs
+  clean: true
+  path: s/API_Specs
+- checkout: SdkGenerator
+  clean: true
+  path: s/SdkGenerator
+- checkout: ${{ parameters.SelfTemplateResource }}
+  clean: true
+  submodules: true
+  path: s/sdks/PythonSDK
+  persistCredentials: true
+- bash: |
+    set -e
+    
+    echo alias the ADO variables into local variables
+    ApiSpecSource="${{ parameters.ApiSpecSource }}"
+    CommitMessage="${{ parameters.CommitMessage }}"
+    GitDestBranch="${{ parameters.GitDestBranch }}"
+    SdkName="${{ parameters.SdkName }}"
+    TestServer="${{ parameters.TestServer }}"
+    WORKSPACE=$(pwd -W)
+    # Hack attempt to get WORKSPACE into a sub-environment
+    export WORKSPACE="$WORKSPACE"
+     
+    . "$WORKSPACE/JenkinsSdkSetupScripts/JenkinsScripts/Pipeline/testInit.sh"
+    cd "$WORKSPACE/SDKGenerator/SDKBuildScripts"
+    export PF_TEST_TITLE_DATA_JSON="$WORKSPACE\JenkinsSdkSetupScripts\Creds\testTitleData.json"
+    . ./shared_build.sh
+    
+    echo === Build the Python Project ===
+    cd $WORKSPACE/sdks/$SdkName/playfab
+    python testCompile.py
+
+    if [ "$isVersioned" = "true" ]; then
+      echo === publish to pypi ===
```

By tabbing I actually meant:
```suggestion
        echo === publish to pypi ===
```
And the same for the rest of this block.